.github/workflows/tests.yaml

```diff
@@ -74,3 +74,20 @@ jobs:
           run: |
             # check if everything executed without errors
             cd scripts/workflow && bash exit_code.sh
+
+    shellcheck:
+      runs-on: ubuntu-latest
+      if: github.event.pull_request.draft == false
+      steps:
+        - uses: actions/checkout@v3
+
+        - uses: ludeeus/action-shellcheck@master
+          env:
+            # Only check "Double quote to prevent globbing and word splitting."
+            # now, but we should check others in the future.
+            # https://github.com/koalaman/shellcheck/wiki/Sc2086
+            SHELLCHECK_OPTS: -i SC2086
+          with:
+            ignore_paths:
```

Comment:
We probably also need to ignore everything in `evaluation` and all other auxiliary directories.

Reply:
OK. I'll add the path to `ignore_paths` after merging https://github.com/a5ob7r/pash/pull/1. In addition, we should add the `scripts` dir to it too?

Reply:
I looked at the ones that fail, and I would suggest adding all of them (`annotations`, `evaluation`, `runtime`, `scripts`). 

We could do another PR on top of this branch of yours so that we check that shellcheck CI passes before we actually merge it here (so that we don't introduce a failing CI workflow).